### ISSUE TITLE
Exported XLS - cell format

### ISSUE THREAD

**Issue by VOOM108** [kevinpapst/kimai2#2744]
**Describe the bug**
When exporting a decimal time into an XLS, the cell is not formatted as a number but as text. Even if it is formatted as a number in the template. Strangely only values without decimal values are a number in the result, anything else displays as text - and can only be converted one by one. Thus, the accounting cannot build a sum from the decimal time values. Of course I could enter a variable with the sum, but they want to calculate this themselves as a validity check. 

![kimai2-xls-exported](https://user-images.githubusercontent.com/16562946/131672756-53da0616-0be8-43e9-87a5-704dfe5e36e8.png)

**Comment by pkaltenboeck** [comment]
Which version / parameters do you use for exporting? Do you use a custom exporter?
Since we never used the excel export till now, I just tried it by myself, but can't confirm the bug. In my excel export the Duration is a excel formula and correctly formated:
![image](https://user-images.githubusercontent.com/1273655/132071417-46817a6e-fea5-43c8-aa96-9763d538cafd.png)

Following system has be used for my screenshot:
- Kimai2 1.15 dev (Ayumi)
- Account with ROLE_SUPER_ADMIN
- "Use decimal format for export" activated

**Comment by VOOM108** [comment]
Here: 
Kimai 1.14.3
Account with ROLE_SUPER_ADMIN
"Use decimal format for export" activated

In your file line 36 - what is the warning in that cell? 

The XLS-Template was uploaded via the GUI... 
[TS_dezimal.xlsx](https://github.com/kevinpapst/kimai2/files/7108813/TS_dezimal.xlsx)

**Comment by pkaltenboeck** [comment]
Ok, you are using a custom template - I just tried it with the default export from kimai. In line 36 it's just the "invalid formula" error which excel sometimes "randomly" has if the formular above differs.
![image](https://user-images.githubusercontent.com/1273655/132073258-f7820576-6d0e-451f-95c6-10c5f8d30818.png)

I just tried it with the provided template and can confirm the problem, but I'm not sure if it's an kimai problem or an excel problem.

I think I found a solution or at least a workaround: Instead of using `${entry.duration_decimal}` use `'=${entry.duration}/3600`
![image](https://user-images.githubusercontent.com/1273655/132074672-57861875-7a05-4ede-92b7-c8fd21df73a6.png)

Be aware of the `'` in front of the formula, otherwise Excel doesn't allow you to confirm your input.

The updated XLS-Template:
[TS_dezimal_updated.xlsx](https://github.com/kevinpapst/kimai2/files/7108902/TS_dezimal_updated.xlsx)

BR
Patrick
